app/component/map/map.cjsx

```diff
@@ -45,6 +46,7 @@ class Map extends React.Component
 
       if @props.showStops
         stops = <StopMarkerContainer hilightedStops={@props.hilightedStops}/>
+        cityBikes = <CityBikeMarkerContainer/>
```

Comment:
The hilightedStops attr relates to stop page, where the icon is hilighted. We can add that conf when we create the stop page for the bikeshare station